packages/libs/sdk-mixins/src/mixins/staticResourcesMixin/staticResourcesMixin.ts

```diff
@@ -46,7 +46,7 @@ export const staticResourcesMixin = createSingletonMixin(
         const resourceUrl = getResourceUrl({
           projectId: this.projectId,
           filename,
-          baseUrl: this.baseUrl,
+          baseUrl: this.baseStaticUrl,
```

Comment:
Can this break compatibility? (From versions prior your last changes)

Reply:
This breaks only the last release which I'm pretty sure no one is currently using